### ISSUE TITLE
Add events to a "bundle" for easier subscription management

### ISSUE THREAD

**Issue by novalore** [Slanfan/MTG-Tolaria#349]
When organizing bigger events on multiple days that include main and side event it would be very useful to allow the organizers to create "bundles" or "groups" of events, especially when there is a discoutn in place for players attending all the events or a combination of them.
For example: TestCon is an event played over an entire weekend, with a tournament on Friday, another one on Saturday, and two smaller side events on Sunday. If they were all bundled together, it would allow the organizers to set a participation price for each tournament and dicounted prices for participation in one event each day.
Is the idea explained well enough?

**Comment by Slanfan** [comment]
Yes. We should discuss how to create some kind of ticket management where you can bundle multiple events in the same ticket and use it for the events in question. As of now I have no idea on how to architect this but I guess we can figure it out together.

**Comment by novalore** [comment]
I agree it would be good to separate ticket management from tournament management. Let's discuss it
